downstream/modules/aap-hardening/con-planning-considerations.adoc

```diff
@@ -7,16 +7,16 @@
 
 [role="_abstract"]
 
-When planning an {PlatformNameShort} installation, ensure that the following components are included:
-
-* Installer-manged components
-** {ControllerNameStart}
-** {EDAcontroller}
-** {PrivateHubNameStart}
-* PostgreSQL database (if not external)
-** External services
-** {InsightsName}
-** {HubNameStart}
-** `registry.redhat.io` (default {ExecEnvShort} container registry)
-
-See the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_planning_guide/platform-system-requirements[system requirements] section of the _{PlatformName} Planning Guide_ for additional information.
+{PlatformName} is composed of the following primary components:
+
+
+
+* {ControllerNameStart}
+* Automation mesh
```

Comment:
We do have attributes for automation mesh: 
:AutomationMesh: automation mesh
:AutomationMeshStart: Automation mesh